aspnetcore/migration/70-80.md

```diff
@@ -73,64 +73,541 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
+
+   ```razor
+   @if (Env.IsDevelopment())
+   {
+       <text>
+           An unhandled exception has occurred. See browser dev tools for details.
+       </text>
+   }
+   else
+   {
+       <text>
+           An error has occurred. This app may no longer respond until reloaded.
+       </text>
+   }
+   ```
+
+   Change the Blazor script from `blazor.server.js` to `blazor.web.js`:
+
+   ```diff
+   - <script src="_framework/blazor.server.js"></script>
+   + <script src="_framework/blazor.web.js"></script>
+   ```
+
+1. Delete the `Pages/_Host.cshtml` file.
+
+1. Update `Program.cs`:
+
+   > [!NOTE]
+   > In the following example, the project's namespace is `Net8BlazorServer`. Adjust the namespace to match your project.
+
+   Add a `using` statement to the top of the file for the project's namespace:
+
+   ```csharp
+   using Net8BlazorServer;
+   ```
+  
+   Replace `AddServerSideBlazor` with `AddRazorComponents` and a chained call to `AddInteractiveServerComponents`.
+   
+   Remove the following line:
+
+   ```diff
+   - builder.Services.AddServerSideBlazor();
+   ```
+
+   Replace the line with the following lines:
+
+   ```csharp
+   builder.Services.AddRazorComponents()
+       .AddInteractiveServerComponents();
+   ```
+
+   Replace `MapBlazorHub` with `MapRazorComponents`, supplying the `App` component as the root component type, and add a chained call to `AddInteractiveServerRenderMode`.
+   
+   Remove the following line:
+  
+   ```diff
+   - app.MapBlazorHub();
+   ```
+
+   Replace the line with the following lines:
+
+   ```csharp
+   app.MapRazorComponents<App>()
+       .AddInteractiveServerRenderMode();
+   ```
+
+   Remove the following line:
+   
+   ```diff
+   - app.MapFallbackToPage("/_Host");
+   ```
+
+   Add [Antiforgery Middleware](xref:blazor/security/index#antiforgery-support) to the request processing pipeline immediately after the call to `app.UseRouting`. If there are calls to `app.UseRouting` and `app.UseEndpoints`, the call to `app.UseAntiforgery` must go between them. Calls to `app.UseAntiforgery` must be placed after calls to `app.UseAuthentication` and `app.UseAuthorization`.
+
+   ```csharp
+   app.UseAntiforgery();
+   ```
+
+1. If the Blazor Server app was configured to disable prerendering, you can continue to disable prerendering for the updated app. In the `App` component, change the value assigned to the `@rendermode` Razor directive attributes for the `HeadOutlet` and `Routes` components.
+
+   Remove the following from the `HeadOutlet` and `Routes` components:
+
+   ```diff
+   - @rendermode="RenderMode.InteractiveServer"
+   ```
+
+   Replace with the following:
+
+   ```razor
+   @rendermode="new InteractiveServerRenderMode(prerender: false)"
+   ```
+
+   For more information, see <xref:blazor/components/render-modes?view=aspnetcore-8.0&preserve-view=true#prerendering>.
+
+### Add interactive WebAssembly components to a Blazor Server app
```

Comment:
As mentioned above, I'd like to move this content to the Integrate components topic.